### PR TITLE
Libnetwork pip installable

### PR DESCRIPTION
**Pull request by gunjan5**
for issue [1154](https://github.com/projectcalico/calico-containers/issues/1154)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import setuptools
+
+version = '0.9.0'
```
Comment (robbrockbank) [inline setup.py]: I think this should be 0.9.0.dev1 or similar (that's in keeping with our master version  convention of Felix.

File: libnetwork.egg-info/PKG-INFO [hunk 2]
```diff
@@ -0,0 +1,18 @@
+Metadata-Version: 1.1
```
Comment (robbrockbank) [inline libnetwork.egg-info/PKG-INFO]: Don't think these egg files should be here

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        'License :: OSI Approved :: Apache Software License',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Topic :: System :: Networking',
+    ],
+
+     # What does your project relate to?
+    keywords='calico docker etcd mesos kubernetes rkt openstack',
+
+    packages=setuptools.find_packages(exclude=['ez_setup', 'examples', 'tests']),
```
Comment (robbrockbank) [inline setup.py]: Can we just set this to be ["libnetwork"]

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    include_package_data=True,
```
Comment (robbrockbank) [inline setup.py]: Should this be here?

File: setup.py (hunk 1 above)
Comment (robbrockbank) [inline setup.py]: or 0.9.0-dev in keeping with libcalico

File: libnetwork.egg-info/PKG-INFO (hunk 2 above)
Comment (gunjan5) [inline libnetwork.egg-info/PKG-INFO]: done!

File: setup.py (hunk 4 above)
Comment (gunjan5) [inline setup.py]: removed!

File: setup.py (hunk 3 above)
Comment (gunjan5) [inline setup.py]: done!

File: setup.py (hunk 1 above)
Comment (gunjan5) [inline setup.py]: done!

**Comment by robbrockbank** [comment]
LGTM - but I'd like to get the nod that you've tested the pip installed code and had it working.

**Comment by matthewdupre** [comment]
@gunjan5 did you get round to doing the test Rob suggested yet?

**Comment by gunjan5** [comment]
@matthewdupre yup! It works, however I wanted to do the whole test again after rebasing/refactoring (the part that is in `calico-containers` repo) once the GoBGP PR [1126](https://github.com/projectcalico/calico-containers/pull/1126) is merged

**Pull request merged**